scylla/src/transport/session_test.rs

```diff
@@ -1397,18 +1399,67 @@ async fn test_schema_types_in_metadata() {
     assert_eq!(
         a.type_,
         CqlType::UserDefinedType {
-            name: "type_a".to_string(),
-            frozen: true
+            frozen: true,
+            definition: Ok(Arc::new(UserDefinedType {
```

Comment:
This is the definition of type "type_a`, and I see that you provided it in two places - here and at line 1595. I think you should only define it once as a variable and then use it in both places.